mp/src/game/server/triggers.cpp

```diff
@@ -386,7 +375,7 @@ bool CBaseTrigger::PassesTriggerFilters(CBaseEntity *pOther)
 
 			if ( HasSpawnFlags( SF_TRIGGER_ONLY_NPCS_IN_VEHICLES ) )
```

Comment:
You may want to just remove these spawn flags altogether, since there's no vehicles anymore anyways. Make sure to edit the base FGD file in the `game/bin/` folder as well!